index.html

```diff
@@ -1246,334 +1093,764 @@ <h3>DID URL Content Metadata</h3>
 		</p>
 	</section>
 
-	<section id="dereferencing-algorithm">
-		<h2>DID URL Dereferencing Algorithm</h2>
-		<p>The following <a>DID URL dereferencing</a> algorithm MUST be implemented by a conformant <a>DID resolver</a>.
-		In accordance with [[RFC3986]], it consists of the following three steps: resolving the DID; dereferencing the
-		resource; and dereferencing the fragment (only if the <var>input <a>DID URL</a></var> contains a <a>DID fragment</a>):</p>
+	<section id="did-url-components">
+		<h3>DID URL Component Handling</h3>
 
-		<section id="dereferencing-algorithm-resource">
-			<h2>Dereferencing the Resource</h2>
-			<ol class="algorithm">
-				<li>Validate that the <var>input <a>DID URL</a></var> conforms to the `did-url` rule of the
-					<a href="https://www.w3.org/TR/did-core/#did-url-syntax">DID URL Syntax</a>.
-					If not, the <a>DID URL dereferencer</a> MUST return the following result:
-					<ol class="algorithm">
-						<li><b>dereferencingMetadata</b>: <var>error object</var> with type set to <a href="#INVALID_DID_URL">INVALID_DID_URL</a></li>
-						<li><b>contentStream</b>: <code>null</code></li>
-						<li><b>contentMetadata</b>: <code>«[ ]»</code></li>
+		<p>
+			A <a>DID URL</a> consists of a <a>DID</a>, optionally followed by
+			additional URL components — a path, query parameters, and a fragment
+			— as defined in
+			<a
+			href="https://www.rfc-editor.org/rfc/rfc3986#section-3">RFC3986</a>.
+			This section defines how these components <strong>MUST</strong> be
```

Comment:
```suggestion
			This section defines how these components MUST be
```

I don't think we want to bold normative statements to maintain consistency with the rest of the spec text.

Reply:
Thanks @wip-abramson -- will remove.  As noted, I have a pure cleanup pass still to go. I should have checked on this one....